configure.ac

```diff
@@ -1296,6 +1302,10 @@ if test "${enable_async_push}" = "yes"; then
 	)
 fi
 
+if test "$VLAN_TAGGING" = "yes"; then
+   AC_DEFINE(ENABLE_VLAN_TAGGING, 1, [Enable 802.1Q-based VLAN tagging/untagging])
+fi
```

Comment:
Without "the knob" present from the start, it would be more difficult to compile test the patches separately - unless the knob-adding patch would also be the one were we add all the `#ifdef`s. Would that be in line with your thinking?

Reply:
I wouldn't recommend so - better to add the `#ifdefs` with the code.

For compile testing I think you can just add the define by specifying `CFLAGS="-DENABLE_VLAN_TAGGING=1"` when launching configure. This is also how we test compile time flags (i.e. -Werror and similar). Wouldn't that work?

Reply:
Yup, good point.